test/test_core.py

```diff
@@ -4813,6 +4813,68 @@ def test_dylink_exceptions_try_catch_6(self):
 
     self.do_runf("main.cpp", "side: caught int 3\n")
 
+  @with_both_eh_sjlj
+  @needs_dylink
+  def test_dylink_exceptions_try_catch_64_bit(self):
+    self.set_setting('WASM_BIGINT')
```

Comment:
Can you run this test both with and without `WASM_BIGINT` enabled?

I'm actually curious to see if it works in the that case.